docs/reset-psw-user.rst

```diff
@@ -0,0 +1,22 @@
+=====================================
+Reset specific user password by shell
+=====================================
+
+In case you don't remember the user password it is easy to do change by CLI. For example:
+
+- Run the shell mode in Django:
+
+   ``docker-compose run --rm web ./manage.py shell``
+
+- After loading you will be in the shell mode where you can run python code:
+
+.. code-block::
+
+    from django.contrib.auth.models import User #import the class
+    users = User.objects.filter()
+    users # will show the list of accounts with their public names
+    user=users[0] # with this example we take the first user
```

Comment:
Be querying with `get` the couple of lines above are not needed (users and user=users[0])